libsoftwarecontainer/component-test/softwarecontainerlib_componenttest.cpp

```diff
@@ -508,28 +522,29 @@ TEST_F(SoftwareContainerApp, TestUnixSocket) {
         sleep(1);
```

Comment:
I would prefer not to touch the tests as such in this PR, actually, but in general I agree.